src/main/java/racingcar/car/NextStepCars.java

```diff
@@ -2,36 +2,64 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Random;
 import java.util.stream.Collectors;
 
 public class NextStepCars {
     private List<Car> nextStepCars;
 
-    public NextStepCars(List<Car> nextStepCars) {
-        this.nextStepCars = nextStepCars;
-    }
-
-    public NextStepCars(String carList) {
+    public NextStepCars(Inputs inputs) {
         nextStepCars = new ArrayList<>();
-        for(String car : carList.split(",")) {
+        for(String car : inputs.getCarNames()) {
             nextStepCars.add(new NextStepCar(car));
         }
     }
 
-    public List<Car> getNextStepCars() {
-        return nextStepCars;
+    public NextStepCars(List<Car> nextStepCars) {
+        this.nextStepCars = nextStepCars;
     }
 
     public int carCount() {
         return nextStepCars.size();
     }
 
-    public List<String> getWinner() {
+    private int getFinalWinnerPosition() {
         int winnerPosition = 0;
         for(Car car : this.nextStepCars) {
             winnerPosition = Math.max(winnerPosition, car.getPosition());
         }
         int finalWinnerPosition = winnerPosition;
+        return finalWinnerPosition;
+    }
+
+    public void move(Random randomNumber) {
+        for(Car car : this.nextStepCars) {
+            car.moveForward(randomNumber.nextInt(10));
+        }
+    }
+
+    public String solveResult() {
+        return this.generateResultString();
+    }
+
+    /**
+     * 출력 해 줄 문자열생성
+     * @return
+     */
+    private String generateResultString() {
+        StringBuilder sb = new StringBuilder();
+        for(Car car : this.nextStepCars) {
+            sb.append(car.getName() + "-".repeat(car.getPosition())+"\n");
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 좋을 것같아요! 
```suggestion
            sb.append(car.getName() + "-".repeat(car.getPosition()) + System.lineSeperator());
```